bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/size/IDFSizeAnalysisEditor.java

```diff
@@ -37,7 +37,8 @@ protected void createPages()
 		String osString = file.getLocation().toOSString();
 		project = file.getProject();
 		Logger.log("Editor input:" + osString); //$NON-NLS-1$
-
+		
+		setEditorTabTitle(project.getName());
```

Comment:
This is good, but the ideal way to set the setPartName(..) is part of the MultiPageEditorPart.init(...) 

You can override the init(...) implementation and then set the part name setPartName(..) directly without calling a different method again